CONTRIBUTING.md

```diff
@@ -224,6 +225,54 @@ GitHub has a [nice introduction](https://help.github.com/articles/github-flow/)
 <img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
 
 
+## Updating the schema
+
+Portions of the BIDS specification are defined using YAML files, in order to make the specification machine-readable.
+Currently, the only portion of the specification that relies on this schema is the Entity Table, but any changes to the specification should be mirrored in the schema.
+
+### The format of the schema
+
+The schema reflects the files and objects in the specification, as well as associations between these objects.
+Here is a list of the files and subfolders of the schema, with a description for each:
+
+- `associated_data.yaml`:
+    Folders that contain files falling outside the jurisdiction of the specification, but which are commonly contained within the same folder as a BIDS dataset.
+    The schema specifies which folders are accepted and whether they are required or optional.
+- `top_level_files.yaml`:
+    Modality-agnostic files stored at the top level of a BIDS dataset.
+    The schema specifies whether these files are required or optional, as well as acceptable extensions for each.
+- `modalities.yaml`:
+    Modalities supported by the specification, along with a list of associated datatypes.
+- `entities.yaml`:
+    A list of entities (key/value pairs in folder and filenames) with associated descriptions and formatting rules.
+    The order of the entities in the file determines the order in which entities must appear in filenames.
+- `datatypes/*.yaml`:
+    Datatypes supported by the specification.
+    Each datatype may support many suffixes.
+    These suffixes are divided into groups based on what extensions and entities are allowed for each.
+- `auxdatatypes/*.yaml`:
```

Comment:
Auxiliary data types also seems like it needs some examples... can't quite guess what this is supposed to be.

Reply:
This, and your other comments, should be addressed now. Thanks!

Reply:
Super!  Looks much clearer now!